explorer/client/src/components/Table/TableCell.tsx

```diff
@@ -25,7 +25,11 @@ interface IProps {
 const renderCol = (col: Column) => {
   switch (col.type) {
     case 'link':
-      return <Link to={col.to}>{col.text}</Link>
+      return col.to.startsWith('http') ? (
+        <a href={col.to}>{col.text}</a>
+      ) : (
+        <Link to={col.to}>{col.text}</Link>
```

Comment:
I feel like this should just be the text. Currently it looks like a link and clears the search in the URL

Reply:
@rupurt Thanks for taking a look. Can you clarify this one? Are you saying that this should not be a clickable link at all or do you mean the visuals are bad? I think the issue here is that this logic currently doesn't check if the `url` is `null`, so the user is deceived into thinking it's a link, when `href` is `null`. I'd like a clarification on `feel like ths should just be the text`.

Reply:
I think if there is no URL for the node it should not be clickable and it should not look like it's clickable i.e. `<Link to={col.to}>{col.text}</Link>` -> `{col.text}`

Reply:
@rupurt Oh of course; I coded this blindly so I couldn't visualize what was going on exactly. I took a look at the story's description again, though, and saw that it never asked for the node names in table cells to be clickable. Story's description only mentions the job run show page.